src-tauri/Cargo.lock

```diff
@@ -3951,9 +3979,9 @@ checksum = "a2eb9349b6444b326872e140eb1cf5e7c522154d69e7a0ffb0fb81c06b37543f"
 
 [[package]]
 name = "steamlocate"
-version = "1.2.1"
```

Comment:
Uh, why is steamlocate being bumped down a version?

Reply:
Also a bunch of other seemingly unrelated package versions changing, maybe reset `Cargo.lock` to version in `main` and then add dependency again?